nextcord/player.py

```diff
@@ -557,7 +557,7 @@ async def probe(
     def _probe_codec_native(
         source, executable: str = "ffmpeg"
     ) -> Tuple[Optional[str], Optional[int]]:
-        exe = executable[:2] + "probe" if executable in ("ffmpeg", "avconv") else executable
+        exe = executable[:-4] + "probe" if executable.endswith(("ffmpeg", "avconv")) else executable
```

Comment:
Just a general question, shouldn't proper path manipulation be used for this?
`Path(executable).parent / "ffprobe"` or similar

Reply:
You're right. I think
```suggestion
        exe = Path(executable).parent / "ffprobe" if Path(executable).name in ("ffmpeg", "avconv") else executable
```
makes the most sense.